clients/vscode/src/chat/WebviewHelper.ts

```diff
@@ -343,6 +343,10 @@ export class WebviewHelper {
     this.client?.sendMessage(message);
   }
 
+  public syncActiveSelectionToChatPanel(context: Context | null) {
+    this.client?.updateActiveSelection(context);
```

Comment:
Please add a try-catch block here to handle the case when connecting to an old version server that does not implement the `updateActiveSelection` API.